model/src/w3updtmd.F90

```diff
@@ -974,6 +1009,17 @@ SUBROUTINE W3UTAU ( FLFRST )
 !/
       END SUBROUTINE W3UTAU
 !/ ------------------------------------------------------------------- /
+!
+!> @author H. L. Tolman @date 06-Jun-2018
```

Comment:
If you consider how we want this documentation to look, you can use the same order in the comments. For example:

https://noaa-emc.github.io/NCEPLIBS-g2c/g2__create_8c.html#a0e8970973e1cefc142240f961a796108

You will see that the order is:

brief description
detailed descrion
parameters
author/date

So the comment above becomes (note that, in this order, the brief and details tags are assumed by doxygen to be the first sentence, and the rest of the text before the parameters).

!> Initialize the wave field with fetch-limited spectra before the
!> actual calculation start.
!> 
!> (Named as an update routine due to placement in code.) Fetch-limited
!> JONSWAP spectra with a cosine^2 directional distribution and a mean direction
!> taken from the wind.
!>
!> @param[out] A Action density spectra
!> @returns A Action density spectra
!> @author H. L. Tolman @date 06-Jun-2018

Reply:
Interesting, I didn't know about that behavior.  I think that's an efficient convention to follow.  I'll make that change.

Reply:
The ordering changes have been made for all the header templates.  As I mention in a comment above I've left the brief and details tags in currently, though have added a note so others are aware they are not necessary.